RussianFilter/sections/specific.txt

```diff
@@ -18,6 +18,7 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+gnatkovsky.com.ua##.r_sell > a[target="_blank"]
```

Comment:
.r_sell is enough in this case.

Reply:
or add domain to `/images/banner-$domain=erospirit.ru|sudba.in.ua`